feature-pack/pom.xml

```diff
@@ -1431,6 +1431,15 @@
             <artifactId>jgroups-azure</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.jgroups.kubernetes</groupId>
+            <artifactId>jgroups-kubernetes</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>net.oauth.core</groupId>
```

Comment:
With the latest version this is no longer necessary. See https://github.com/jgroups-extras/jgroups-kubernetes/commit/08e6455290cae780ef26e07aa593e849bcdf2f2f